README.md

```diff
@@ -42,15 +59,22 @@ Ensure that Python directory is included in your `PATH` system environment varia
 4. Select the serial port associated with the board that you flashed the firmware on.
 5. Click 'Start'.
 
-## Custom Wireshark dissector
+## Wireshark configuration
+
+### Capturing Thread packets
+
+In order to capture packets exchanged on the Thread network it is necessary to configure the Wireshark to use correct decryption keys.
+To set decryption keys go to `Edit -> Preferences... -> Protocols -> IEEE 802.15.4 -> Decryption Keys`. The default decryption key used by nRF5 SDK for Thread and Zigbee examples is `00112233445566778899aabbccddeeff`. Set decryption key index to `0` and `Key hash` to `Thread hash` value.
+
```

Comment:
I think more steps are required.

1. Set up Master Key (you have that one)
2. Thread CoAP - should be set to 61631 i believe (need to verify)
3.  Set up 6LoWPAN contextes (Context 0: fdde:ad00:beef:0::/64, Context 1: fd11:22::/64)
4. Disable ZigBee NWK protocol (and any other unwanted protocols):
Click Analyze and then select Enabled Protocols and Uncheck the following: 
- LwMesh 
- ZigBee
- ZigBee Green Power

Reply:
It's required by Thread Certification. Certain messages are incorrectly interpreted as LwMesh or Zigbee Green Power :)

Reply:
We should probably have a look at the heuristics for LwMesh and ZigBee regarding this packets. Please send me a capture file with incorrectly dissected packets and I'll have a look.

Reply:
@stig-bjorlykke Thank you for offering help. If i will find time i will send you some captures.